src/modules/item/types.ts

```diff
@@ -32,6 +32,8 @@ export enum WearableCategory {
   TOP_HEAD = 'top_head'
 }
 
+export const BODY_SHAPE_CATEGOTY = 'body_shape'
```

Comment:
`BODY_SHAPE_CATEGOTY`

body shape cate(game of the year)

Reply:
xD fixed